rapidast.py

```diff
@@ -154,15 +154,27 @@ def run():
         f"log level set to debug. Config file: '{parser.parse_args().config_file}'"
     )
 
+    # Load config file
     try:
         config = configmodel.RapidastConfigModel(
             yaml.safe_load(load_config_file(parser.parse_args().config_file))
         )
     except yaml.YAMLError as exc:
         raise RuntimeError(
-            f"Something went wrong while parsing one of the config '{parser.parse_args().config_file}':\n {str(exc)}"
+            f"YAML error in config {parser.parse_args().config_file}':\n {str(exc)}"
         ) from exc
 
+    # Optionally adds default if file exists (will not overwrite existing entries)
+    default_conf = os.path.join(os.path.dirname(__file__), "rapidast-defaults.yaml")
```

Comment:
nice approach! just one thing, what do you think about '.rapidast-default.yaml' vs 'rapidast-default.yaml'?